packages/website/plugins/generated-rule-docs.ts

```diff
@@ -1,17 +1,23 @@
 import type * as unist from 'unist';
-import type * as mdast from 'mdast';
+import * as mdast from 'mdast';
+import { format } from 'prettier';
 import type { Plugin } from 'unified';
+import { JSONSchema4Object } from 'json-schema';
+import { compile } from 'json-schema-to-typescript';
 
+import * as tseslintParser from '@typescript-eslint/parser';
 import * as eslintPlugin from '@typescript-eslint/eslint-plugin';
+import { EOL } from 'os';
 
 const generatedRuleDocs: Plugin = () => {
-  return (root, file) => {
+  return async (root, file) => {
     if (file.stem == null) {
       return;
     }
 
-    const docs = eslintPlugin.rules[file.stem]?.meta.docs;
-    if (!docs) {
+    const rule = eslintPlugin.rules[file.stem];
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/pull/5386#issuecomment-1196355572:

> are we planning to add description fields to schema of rules, for describing specific options?

https://github.com/typescript-eslint/typescript-eslint/pull/5386#issuecomment-1197496484:

> It would be a decent chunk of work to add the descriptions for everything up-front.

@armano2 @bradzacher yeah agreed, we should eventually do that. Filed #5392.

For this PR I can go through and move descriptions into rules like [`explicit-function-return-type`](https://deploy-preview-5386--typescript-eslint.netlify.app/rules/explicit-function-return-type#options) where there would be duplicated docs.

Reply:
Ok, I've gone through the rules and think I've gotten all the redundant text out.